### ISSUE TITLE
missing tags on https://pkg.go.dev/

### ISSUE THREAD

**Issue by antoine-choimet-cbp** [appscode/jsonpatch#34]
Hello,

It appears that your newest tags (after 1.0.1) are now missing on pkg.go.dev. 

It break the builds of another projects like cert-manager.

Do you know why your v2 tags are missing?

https://gomodules.xyz/jsonpatch/v2   404

**Comment by Rakshith-R** [comment]
> Hello,
> 
> It appears that your newest tags (after 1.0.1) are now missing on pkg.go.dev.
> 
> It break the builds of another projects like cert-manager.
> 
> Do you know why your v2 tags are missing?
> 
> https://gomodules.xyz/jsonpatch/v2 404


Similar incident is happening at https://github.com/ceph/ceph-csi/ 
https://github.com/ceph/ceph-csi/runs/3460028968?check_suite_focus=true
Thanks in advance! 

cc @tamalsaha

**Comment by capri-xiyue** [comment]
similar issue is happening at https://github.com/kubeflow/pipelines

**Comment by tamalsaha** [comment]
Sorry about the issue. I switched over our GO vanity website to an appengine based site https://github.com/gomodules/govanityurls-appengine to a GoHugo based site https://github.com/gomodules/govanityurls over the weekend. Since Google launched https://deps.dev, our appengine bill went from $0/month to around $100/month just to serve these govanity sites. So, we have switched over to GoHugo and Firebase option.

I followed https://blog.jbowen.dev/2020/07/using-go-vanity-urls-with-hugo/ post to get this GoHugo site.  
I am going to check what is the correct response to return for v2 tag. :man_facepalming:
